testcases/misc_tests/cca_export_import_test.c

```diff
@@ -1259,6 +1280,15 @@ static CK_RV cca_rsa_export_import_tests(void)
         priv_opaquekey = NULL;
         free(publ_opaquekey);
         publ_opaquekey = NULL;
+
+        if (publ_key != CK_INVALID_HANDLE)
+            funcs->C_DestroyObject(session, publ_key);
```

Comment:
Hm, minor issue for those cases that run inside a loop.

If the first iteration works fine, and thus destroyed the objects, then the handles are still left at at a now invalid handle value. If the next iteration fails and goes to the error label early, then it tries to destroy already destroyed objects.  Should not really hurt, C_DestroyObject will silently fail ....

Reply:
Hmm, yes that's right ... similar to setting buffers to NULL after a free to avoid double-free's, we could add a xxx_key = CK_INVALID_HANDLE after the C_DestroyObject ...

Reply:
Done